libafl_frida/Cargo.toml

```diff
@@ -27,11 +27,11 @@ hashbrown = "0.11"
 libloading = "0.7.0"
 rangemap = "0.1.10"
 frida-gum-sys = { version = "0.3", features = [ "auto-download", "event-sink", "invocation-listener"] }
-frida-gum = { version = "0.5.2", features = [ "auto-download", "event-sink", "invocation-listener"] }
+frida-gum = { version = "0.6.0", features = [ "auto-download", "event-sink", "invocation-listener"], git = "https://github.com/frida/frida-rust" }
```

Comment:
Do we really want the git reference?

Reply:
It should be. I'm using it in other stuff, IIRC.

Reply:
The updated one is not on crates.io yet.
https://github.com/frida/frida-rust/pull/29#issuecomment-951469179

Reply:
I've published 0.6.1 which includes these changes. Thanks.

Reply:
thank you!
I'll update cargo.toml